src/lib/ngsiNotify/senderThread.cpp

```diff
@@ -114,16 +133,37 @@ void* startSenderThread(void* p)
       {
         __sync_fetch_and_add(&noOfNotificationsSent, 1);
         alarmMgr.notificationErrorReset(url);
+        params->failsCounter = 0;
 
         if (params->registration == false)
         {
           subNotificationErrorStatus(params->tenant, params->subscriptionId, 0, statusCode, "");
         }
       }
+
+
       else
       {
         alarmMgr.notificationError(url, "notification failure for sender-thread: " + out);
 
+        params->failsCounter++;
```

Comment:
Similar comments to [the ones provided for `workerFunc()`](https://github.com/telefonicaid/fiware-orion/pull/3890/files#r709082649) also apply here.

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3890/commits/6b68a8c9ded45dd2e30c033f4d5000cddfab19de